packages/foam-vscode/src/features/copy-without-brackets.spec.ts

```diff
@@ -1,21 +1,18 @@
-// import { env, window, Uri, Position, Selection, commands } from 'vscode';
-// import * as vscode from 'vscode';
+import { env, Position, Selection, commands } from 'vscode';
+import {
+  createFile,
+  getUriInWorkspace,
+  showInEditor,
+} from '../test/test-utils-vscode';
 
 describe('copyWithoutBrackets', () => {
-  it('should pass CI', () => {
-    expect(true).toBe(true);
+  it('should get the input from the active editor selection', async () => {
+    await createFile('This is my [[test-content]].', ['hello.md']);
+    const uri = getUriInWorkspace('hello.md');
```

Comment:
One thing I missed before, you should be able to do this:

```suggestion
    const { uri } = await createFile('This is my [[test-content]].');
```

Also, what was causing the random fail in the test and in what way was the issue fixed?

Reply:
Thanks!

As regarding the cause, when i was running tests with previous code, I notice that sometimes other test will intercept this `should get the input from the active editor selection` test, for example:

```
 FAIL  src/dated-notes.spec.ts
  ● Daily note template › Uses the daily note variables in the template

    expect(received).toEqual(expected) // deep equality

    - Expected  - 0
    + Received  + 1

      hello September 12 hello
    + This is my [[test-content]].

      86 |     const doc = await showInEditor(uri);
      87 |     const content = doc.editor.document.getText();
    > 88 |     expect(content).toEqual('hello September 12 hello');
         |                     ^
      89 |   });
      90 |
      91 |   afterAll(async () => {

      at Object.<anonymous> (src/dated-notes.spec.ts:88:21)
```

this occurs in one of my runs, where this unrelated test received `This is my [[test-content]].`.

I rewrite the likely offending lines and the failures no longer occur.